src/sentry/coreapi.py

```diff
@@ -328,6 +328,21 @@ def _process_fingerprint(self, data):
             result.append(unicode(bit))
         return result
 
+    def parse_client_as_sdk(self, value):
+        if not value:
+            return
+        try:
+            name, version = value.split('/', 1)
+        except ValueError:
+            try:
+                name, verison = value.split(' ', 1)
```

Comment:
Also, why is space a valid delimiter here? Is it allowed to be `raven-js 2.0.0`? instead of `raven-js/2.0.0`?